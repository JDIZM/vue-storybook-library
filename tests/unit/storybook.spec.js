// tests/unit/storybook.spec.js
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots();
