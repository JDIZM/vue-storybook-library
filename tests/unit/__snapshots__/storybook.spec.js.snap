// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots atoms/Button Primary 1`] = `
<div
  class="btn btn--primary"
>
  
  Button

</div>
`;

exports[`Storyshots atoms/Button Secondary 1`] = `
<div
  class="btn btn--secondary"
>
  
  Button

</div>
`;

exports[`Storyshots atoms/Input Text Field 1`] = `
<div
  class="form__field"
>
  <label
    for="TextField"
  >
    Input
  </label>
   
  <!---->
   
  <input
    class="bg--primary-100"
    id="TextField"
    name="TextField"
    placeholder="Enter some text"
    type="text"
  />
</div>
`;

exports[`Storyshots atoms/Input Text Field With Error 1`] = `
<div
  class="form__field --error"
>
  <label
    for="TextFieldWithError"
  >
    Input
  </label>
   
  <span
    class="material-icons"
  >
    phone
  </span>
   
  <input
    class="bg--primary-100"
    id="TextFieldWithError"
    name="TextFieldWithError"
    placeholder="Enter your phone number"
    type="tel"
  />
</div>
`;

exports[`Storyshots atoms/Input Text Field With Icon 1`] = `
<div
  class="form__field"
>
  <label
    for="TextFieldWithIcon"
  >
    Input
  </label>
   
  <span
    class="material-icons"
  >
    phone
  </span>
   
  <input
    class="bg--primary-100"
    id="TextFieldWithIcon"
    name="TextFieldWithIcon"
    placeholder="Enter your phone number"
    type="tel"
  />
</div>
`;

exports[`Storyshots molecules/FormField Input 1`] = `
<div
  class="form__field"
>
  <label
    for="Input"
  >
    Input
  </label>
   
  <span
    class="material-icons"
  >
    phone
  </span>
   
  <input
    class="bg--primary-100"
    id="Input"
    name="Input"
    placeholder="Enter your phone number"
    type="tel"
  />
</div>
`;

exports[`Storyshots molecules/FormField Input With Error 1`] = `
<div
  class="form__field --error"
>
  <label
    for="InputWithError"
  >
    Input
  </label>
   
  <span
    class="material-icons"
  >
    phone
  </span>
   
  <input
    class="bg--primary-100"
    id="InputWithError"
    name="InputWithError"
    placeholder="Enter your phone number"
    type="tel"
  />
</div>
`;

exports[`Storyshots molecules/FormField Input With Icon 1`] = `
<div
  class="form__field"
>
  <label
    for="InputWithIcon"
  >
    Input
  </label>
   
  <span
    class="material-icons"
  >
    phone
  </span>
   
  <input
    class="bg--primary-100"
    id="InputWithIcon"
    name="InputWithIcon"
    placeholder="Enter your phone number"
    type="tel"
  />
</div>
`;

exports[`Storyshots organisms/Footer Primary 1`] = `
<footer
  class="bg--primary text--white"
>
  <div
    class="footer"
  >
    <div
      class="footer__grid"
    >
      <div
        class="footer__grid__item brand"
      >
        <div
          class="brand__logo"
        >
          <img
            draggable="false"
            src=""
          />
        </div>
         
        <div
          class="brand__name"
        />
      </div>
       
      <div
        class="footer__grid__item social"
      >
        <h3>
          SOCIAL
        </h3>
         
        <ul>
          <li>
            <a
              aria-label="Find us on LinkedIn"
              href="https://linkedin.com"
            >
              <i
                class="fab fa-linkedin"
              />
            </a>
          </li>
          <li>
            <a
              aria-label="Find us on Facebook"
              href="https://facebook.com"
            >
              <i
                class="fab fa-facebook-square"
              />
            </a>
          </li>
          <li>
            <a
              aria-label="Find us on Twitter"
              href="https://twitter.com"
            >
              <i
                class="fab fa-twitter"
              />
            </a>
          </li>
          <li>
            <a
              aria-label="Find us on Github"
              href="https://github.com"
            >
              <i
                class="fab fa-github"
              />
            </a>
          </li>
        </ul>
      </div>
       
      <div
        class="footer__grid__item info"
      >
        <h3>
          COMPANY INFO
        </h3>
         
        <p>
          Company Name
        </p>
         
        <p>
          
          349 Manchester Road 
          <br />
          
          Manchester 
          <br />
          
          M12 ABC
        
        </p>
      </div>
    </div>
     
    <div
      class="copy"
    >
      <p>
        
        Â© Company Name - Company number 12345678 -
        
        <a
          href="/privacy.pdf"
        >
          Privacy
        </a>
      </p>
    </div>
  </div>
</footer>
`;

exports[`Storyshots organisms/Form Primary 1`] = `
<div
  class="form"
  primary="true"
/>
`;

exports[`Storyshots organisms/Nav Primary 1`] = `
<nav
  class="nav"
>
  <div
    class="nav__desktop"
  >
    <div
      class="nav__desktop__brand"
    >
      <div
        class="nav__desktop__logo"
      />
    </div>
     
    <ul
      class="nav__desktop__list"
    >
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/about/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
      <li>
        <a
          aria-pressed="false"
          exact-active-class="--active"
          role="button"
          tabindex="0"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              DROPDOWN
            </span>
          </div>
        </a>
      </li>
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/contact/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
    </ul>
     
    <div
      class="nav__desktop__btns"
    >
      <a
        class="nav__desktop__btn btn--secondary"
        href="tel:undefined"
      >
        
        CALL
      
      </a>
       
      <div
        aria-pressed="false"
        class="nav__desktop__btn btn--primary"
        role="button"
        tabindex="0"
      >
        
        ENQUIRE
      
      </div>
    </div>
  </div>
   
  <div
    class="nav__mobile"
  >
    <div
      class="nav__mobile__brand"
    >
      <div
        class="nav__mobile__logo"
      />
    </div>
     
    <div
      class="nav__mobile__btns"
    >
      <a
        class="nav__mobile__btn btn--secondary"
        href="tel:12345678"
      >
        
        CALL
      
      </a>
       
      <div
        aria-pressed="true"
        class="nav__mobile__btn btn--primary"
        role="button"
        tabindex="0"
      >
        
        ENQUIRE
      
      </div>
    </div>
  </div>
   
  <div
    aria-pressed="true"
    class="nav__toggle"
    role="button"
    tabindex="0"
  >
    <i
      class="material-icons"
    >
      menu
    </i>
  </div>
   
  <!---->
   
  <!---->
</nav>
`;

exports[`Storyshots organisms/Nav Show Drawer 1`] = `
<nav
  class="nav"
>
  <div
    class="nav__desktop"
  >
    <div
      class="nav__desktop__brand"
    >
      <div
        class="nav__desktop__logo"
      />
    </div>
     
    <ul
      class="nav__desktop__list"
    >
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/about/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
      <li>
        <a
          aria-pressed="false"
          exact-active-class="--active"
          role="button"
          tabindex="0"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              DROPDOWN
            </span>
          </div>
        </a>
      </li>
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/contact/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
    </ul>
     
    <div
      class="nav__desktop__btns"
    >
      <a
        class="nav__desktop__btn btn--secondary"
        href="tel:undefined"
      >
        
        CALL
      
      </a>
       
      <div
        aria-pressed="false"
        class="nav__desktop__btn btn--primary"
        role="button"
        tabindex="0"
      >
        
        ENQUIRE
      
      </div>
    </div>
  </div>
   
  <div
    class="nav__mobile"
  >
    <div
      class="nav__mobile__brand"
    >
      <div
        class="nav__mobile__logo"
      />
    </div>
     
    <div
      class="nav__mobile__btns"
    >
      <a
        class="nav__mobile__btn btn--secondary"
        href="tel:12345678"
      >
        
        CALL
      
      </a>
       
      <div
        aria-pressed="true"
        class="nav__mobile__btn btn--primary"
        role="button"
        tabindex="0"
      >
        
        ENQUIRE
      
      </div>
    </div>
  </div>
   
  <div
    aria-pressed="true"
    class="nav__toggle"
    role="button"
    tabindex="0"
  >
    <i
      class="material-icons"
    >
      menu
    </i>
  </div>
   
  <div
    class="nav__drawer"
  >
    <div
      class="nav__drawer__wrapper"
    >
      <div
        class="nav__drawer__top"
      >
        <div
          class="nav__drawer__brand"
        >
          <div
            class="nav__drawer__logo"
          />
        </div>
         
        <div
          aria-pressed="false"
          class="btn"
          role="button"
          tabindex="0"
        >
          <i
            class="material-icons"
          >
            close
          </i>
        </div>
      </div>
       
      <ul
        class="nav__drawer__list"
      >
        <li>
          <nuxt-link
            exact-active-class="--active"
            to="/"
          >
            <div
              class="nav__menu__wrapper"
            >
              <!---->
               
              <span>
                Item
              </span>
            </div>
          </nuxt-link>
        </li>
        <li>
          <nuxt-link
            exact-active-class="--active"
            to="/about/"
          >
            <div
              class="nav__menu__wrapper"
            >
              <!---->
               
              <span>
                Item
              </span>
            </div>
          </nuxt-link>
        </li>
        <li>
          <a
            aria-pressed="false"
            exact-active-class="--active"
            role="button"
            tabindex="0"
          >
            <div
              class="nav__menu__wrapper"
            >
              <!---->
               
              <span>
                Dropdown
              </span>
            </div>
             
            <ul
              class="nav__drawer__dropdown"
            >
              <li>
                <nuxt-link
                  exact-active-class="--active"
                  to="/domestic-scaffolding/"
                >
                  <div
                    class="nav__drawer__dropdown"
                  >
                    Domestic
                  </div>
                </nuxt-link>
              </li>
              <li>
                <nuxt-link
                  exact-active-class="--active"
                  to="/commercial-scaffolding/"
                >
                  <div
                    class="nav__drawer__dropdown"
                  >
                    Commercial
                  </div>
                </nuxt-link>
              </li>
              <li>
                <nuxt-link
                  exact-active-class="--active"
                  to="/industrial-scaffolding/"
                >
                  <div
                    class="nav__drawer__dropdown"
                  >
                    Industrial
                  </div>
                </nuxt-link>
              </li>
            </ul>
          </a>
        </li>
        <li>
          <nuxt-link
            exact-active-class="--active"
            to="/contact/"
          >
            <div
              class="nav__menu__wrapper"
            >
              <!---->
               
              <span>
                Item
              </span>
            </div>
          </nuxt-link>
        </li>
      </ul>
    </div>
     
    <div
      class="nav__drawer__overlay"
    />
  </div>
   
  <!---->
</nav>
`;

exports[`Storyshots organisms/Nav Show Dropdown 1`] = `
<nav
  class="nav"
>
  <div
    class="nav__desktop"
  >
    <div
      class="nav__desktop__brand"
    >
      <div
        class="nav__desktop__logo"
      />
    </div>
     
    <ul
      class="nav__desktop__list"
    >
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/about/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
      <li>
        <a
          aria-pressed="false"
          exact-active-class="--active"
          role="button"
          tabindex="0"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              DROPDOWN
            </span>
          </div>
        </a>
      </li>
      <li>
        <nuxt-link
          exact-active-class="--active"
          to="/contact/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span>
              ITEM
            </span>
          </div>
        </nuxt-link>
      </li>
    </ul>
     
    <div
      class="nav__desktop__btns"
    >
      <a
        class="nav__desktop__btn btn--secondary"
        href="tel:undefined"
      >
        
        CALL
      
      </a>
       
      <div
        aria-pressed="false"
        class="nav__desktop__btn btn--primary"
        role="button"
        tabindex="0"
      >
        
        ENQUIRE
      
      </div>
    </div>
  </div>
   
  <div
    class="nav__mobile"
  >
    <div
      class="nav__mobile__brand"
    >
      <div
        class="nav__mobile__logo"
      />
    </div>
     
    <div
      class="nav__mobile__btns"
    >
      <a
        class="nav__mobile__btn btn--secondary"
        href="tel:12345678"
      >
        
        CALL
      
      </a>
       
      <div
        aria-pressed="true"
        class="nav__mobile__btn btn--primary"
        role="button"
        tabindex="0"
      >
        
        ENQUIRE
      
      </div>
    </div>
  </div>
   
  <div
    aria-pressed="true"
    class="nav__toggle"
    role="button"
    tabindex="0"
  >
    <i
      class="material-icons"
    >
      menu
    </i>
  </div>
   
  <!---->
   
  <div
    class="nav__dropdown bg--primary"
  >
    <ul>
      <li
        class="nav__dropdown__item"
      >
        <a
          exact-active-class="--active"
          to="/domestic-scaffolding/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span
              class="text--white"
            >
              DOMESTIC
            </span>
          </div>
        </a>
      </li>
      <li
        class="nav__dropdown__item"
      >
        <a
          exact-active-class="--active"
          to="/commercial-scaffolding/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span
              class="text--white"
            >
              COMMERCIAL
            </span>
          </div>
        </a>
      </li>
      <li
        class="nav__dropdown__item"
      >
        <a
          exact-active-class="--active"
          to="/industrial-scaffolding/"
        >
          <div
            class="nav__desktop__list__item"
          >
            <!---->
             
            <span
              class="text--white"
            >
              INDUSTRIAL
            </span>
          </div>
        </a>
      </li>
    </ul>
  </div>
</nav>
`;

exports[`Storyshots pages/NotFoundComponent Primary 1`] = `
<div
  class="notfound"
>
  <h2>
    HELLO, IS IT ME YOUR LOOKING FOR?
  </h2>
</div>
`;
